src/carousel.tsx

```diff
@@ -323,20 +161,11 @@ const BaseCarousel: React.FC<BaseCarouselProps> = ({
     onPageChange,
     dataAttributes,
 }) => {
-    const {texts} = useTheme();
-    const containerType = useContainerType();
+    const {texts, platformOverrides} = useTheme();
     const itemsPerPageConfig = normalizeItemsPerPage(itemsPerPage);
     const mobilePageOffsetConfig = normalizeMobilePageOffset(mobilePageOffset);
-    const {isDesktopOrBigger, isLargeDesktop} = useScreenSize();
+    const {isDesktopOrBigger} = useScreenSize();
```

Comment:
this cant be removed?

Reply:
no, because we need it for the itemsperpage config